liquigraph-cli/src/main/java/org/liquigraph/cli/LiquigraphCli.java

```diff
@@ -121,6 +139,21 @@ public static void main(String[] args) {
         );
     }
 
+    private static void printVersion() {
+        InputStream propsIs = LiquigraphCli.class.getResourceAsStream("/liquigraph-cli.properties");
+        String version = null;
+        if (propsIs != null) {
+            Properties props = new Properties();
+            try {
+                props.load(propsIs);
+            } catch (IOException e) {
+                LOGGER.error("An exception occurred while loading the properties", e);
+            }
+            version = props.getProperty("liquigraph.version");
```

Comment:
I hesitated to do it while implementing, as it seemed a bit overkill in this case: the only thing remaining if I move the property extraction is `System.out.println()`. It's no big deal anyway, so I'll do it.